src/compiler/crystal/compiler.cr

```diff
@@ -384,6 +384,7 @@ module Crystal
         [{cl, cmd, nil}]
       elsif program.has_flag? "wasm32"
         link_flags = @link_flags || ""
+        link_flags += " --stack-first -z stack-size=#{8 * 1024 * 1024}"
```

Comment:
I'm wondering if we should reuse `Fiber::StackPool::STACK_SIZE` or perhaps introduce a more generic property for this. It would also be great to allow configuration of the stack size at compile time.
Perhaps a generic `CRYSTAL_STACK_SIZE` environment variable could be useful for this? It would populate this value here and `Fiber::StackPool::STACK_SIZE` and have 8MB as default value.
Just thinking ahead, this can be implemented outside this PR. (It's probably better to extract it).

Reply:
I like having a `CRYSTAL_STACK_SIZE` environment variable! But let's do that on another PR.